src/main/java/frc/robot/commands/CargoCommands/CargoBucketDown.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.commands.CargoCommands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+/** 
+ * Lowers hatch grabber
```

Comment:
All the comments hear say "hatch" when this is bucket.  Of course this may be moot if bucket is in the trash bucket.

Reply:
agreed